op-conductor/conductor/service.go

```diff
@@ -683,18 +687,27 @@ func (oc *OpConductor) transferLeader() error {
 }
 
 func (oc *OpConductor) stopSequencer() error {
-	oc.log.Info("stopping sequencer", "server", oc.cons.ServerID(), "leader", oc.leader.Load(), "healthy", oc.healthy.Load(), "active", oc.seqActive.Load())
-
-	_, err := oc.ctrl.StopSequencer(context.Background())
-	if err != nil {
-		if strings.Contains(err.Error(), driver.ErrSequencerAlreadyStopped.Error()) {
-			oc.log.Warn("sequencer already stopped.", "err", err)
+	oc.log.Info(
+		"stopping sequencer",
+		"server", oc.cons.ServerID(),
+		"leader", oc.leader.Load(),
+		"healthy", oc.healthy.Load(),
+		"active", oc.seqActive.Load())
+
+	// Getting stuck stopping a sequencer can't be good. Is it okay to fail to stop a sequencer on
+	// shutdown? From what I can tell it is.
```

Comment:
Good to loop in an op-conductor expert to answer this question. My gut feeling is that it's more important to shut down a sequencer rather than shutting down op-conductor quickly, since it's always better to have 0 instead of >1 sequencers 😅

Reply:
So stopSequencer is called after conductor loses leadership. In the event that the stopSequencer call fails, it actually has little real consequences because the sequencer cant produce a block and commit it to the raft log (requires leadership). Once conductor comes back up it will check its leader and sequencer state and attempt to stop the sequencer again.

So it is "okay" to fail to stop a sequencer in this case.

Reply:
Thanks! In that case it makes sense to pass a regular context in here and prevent the conductor getting stuck on an API call (like for instance during shutdown).

Reply:
I've incorporated your very useful comment into the PR.